pkg/healthcheck/healthcheck_output.go

```diff
@@ -233,21 +240,22 @@ func runChecksTable(wout io.Writer, hc Runner, output string) bool {
 
 	var success bool
 	switch output {
-	case "short":
+	case ShortOutput:
 		success = hc.RunChecks(prettyPrintResultsShort)
-	case "extension-short":
+	case ExtensionShortOutput:
 		success = hc.RunChecks(prettyPrintResultsExtensionShort)
 	default:
 		success = hc.RunChecks(prettyPrintResults)
 	}
 
-	// this empty line separates final results from the checks list in the output
-	fmt.Fprintln(wout, "")
-
-	if !success {
-		fmt.Fprintf(wout, "Status check results are %s\n", failStatus)
-	} else {
-		fmt.Fprintf(wout, "Status check results are %s\n", okStatus)
+	if output != ExtensionShortOutput {
```

Comment:
After removing `ExtensionShortOutput`

```suggestion
	if output != ShortOutput {
```